esx_service/cli/vmdkops_admin.py

```diff
@@ -248,13 +253,14 @@ def commands():
                                 }
                             }
                         },
+
                         'ls': {
                             'help': "list VMs in a tenant",
                             'func': tenant_vm_ls,
                             'args': {
                                 '--name': {
                                     'help': "Tenant to list the VMs for",
-                                    'required': False
+                                    'required': True
```

Comment:
Can't this remain false and hence allow users to use a single command to list VMs in all tenants vs doing a) tenant ls and then b) for <all tenants>; do tenant vm ls --name <tenant-name>; done?